pkg/autodiscover/autodiscover_sriov.go

```diff
@@ -3,17 +3,32 @@ package autodiscover
 import (
 	"context"
 
-	sriovNetworkOp "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
 	"github.com/redhat-best-practices-for-k8s/certsuite/internal/clientsholder"
 	kerrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 )
 
-func getSriovNetworks(client *clientsholder.ClientsHolder, namespaces []string) (sriovNetworks []sriovNetworkOp.SriovNetwork, err error) {
-	var sriovNetworkList []sriovNetworkOp.SriovNetwork
+// SriovNetworkGVR defines the GroupVersionResource for SriovNetwork
+var SriovNetworkGVR = schema.GroupVersionResource{
+	Group:    "sriovnetwork.openshift.io",
```

Comment:
Question: Was this always the group, even those this repo/dep wasn't coming from the OCP org?

Reply:
Yes, that's what I've seen in their repo as their group.

Reply:
Odd...but okay! Thanks for clarifying.